app/models/metric/targets.rb

```diff
@@ -70,14 +70,10 @@ def self.capture_storage_targets(hosts)
     hosts.flat_map(&:storages).uniq.select { |s| Storage.supports?(s.store_type) & s.perf_capture_enabled? }
   end
 
-  def self.capture_vm_targets(targets, parent_class, options)
-    enabled_parents = targets.select do |t|
-      t.kind_of?(parent_class) &&
-        t.kind_of?(Metric::CiMixin) &&
-        t.perf_capture_enabled? &&
-        t.respond_to?(:vms)
```

Comment:
> Odd that it calls host.perf_capture_enabled? when it had sophisticated logic before to check ems_cluster for those cases.

@kbrock The way it used to work in that case was that if a cluster is enabled, then all hosts under it are considered enabled...if a cluster is not enabled, then we only take hosts are are directly enabled.

Psudeocodishly, I would expect it to do something like `host.perf_capture_enabled? || host.ems_cluster.perf_capture_enabled?`